.azure/functionApp/slackNotifier.bicep

```diff
@@ -0,0 +1,127 @@
+param location string
+param applicationInsightsName string
+param namePrefix string
+param keyVaultName string
+
+// Storage account names only supports lower case and numbers
+var storageAccountName = '${replace(namePrefix, '-', '')}slacknotifiersa'
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2023-01-01' = {
+    name: storageAccountName
+    location: location
+    sku: {
+        name: 'Standard_LRS'
+    }
+    kind: 'Storage'
+    properties: {
+        supportsHttpsTrafficOnly: true
+        defaultToOAuthAuthentication: true
+    }
+}
+
+resource applicationServicePlan 'Microsoft.Web/serverfarms@2021-03-01' = {
+    name: '${namePrefix}-slacknotifier-asp'
+    location: location
+    sku: {
+        name: 'Y1'
+        tier: 'Dynamic'
+    }
+    properties: {}
+}
+
+resource applicationInsights 'Microsoft.Insights/components@2020-02-02' existing = {
+    name: applicationInsightsName
+}
+
+var functionAppName = '${namePrefix}-slacknotifier-fa'
+resource functionApp 'Microsoft.Web/sites@2023-01-01' = {
+    name: functionAppName
+    location: location
+    kind: 'functionapp'
+    identity: {
+        type: 'SystemAssigned'
+    }
+    properties: {
+        serverFarmId: applicationServicePlan.id
+        publicNetworkAccess: 'Enabled'
+        siteConfig: {
+            // Setting/updating appSettings in separate module in order no not delete already deployed functions, see below
```

Comment:
to not delete*

Reply:
Blir allerede deployed funksjoner slettet om vi har med siteConfig?

Reply:
Hvis vi setter appSettings-objektet (linje 54) inn her, så blir de sletta ja. 
Derfor hentes alle settings ut (linje 68) og merges med de vi har definert her i bicep.